content/docs/api_v2/api_v2_reference.apib

```diff
@@ -433,6 +433,7 @@ The HTTP request must have the `Content-Type: application/json` HTTP header!
 The actor needs to define at least one version of the source code.
 For more information, see [Version object](#reference/actors/version-object).
 
+If you want to make your actor public using `isRequired: true`, you will need to provide the `categories` under which that actor will be classified in Apify Store. [Visit the Store](https://apify.com/store) to see list of available categories. You will also need to add the actor's [`title`](https://docs.apify.com/actors/publishing#title).
```

Comment:
I would point to the list of category constants that the users need to use as well: https://github.com/apify/apify-shared-js/blob/2d43ebc41ece9ad31cd6525bd523fb86939bf860/packages/consts/src/consts.ts#L452-L471

Reply:
@fnesveda these are public, right?